src/isomorphic/ReactDebugTool.js

```diff
@@ -10,7 +10,7 @@
  */
 
 'use strict';
-
+var ReactInvalidSetStateWarningDevTool = require('./devtools/ReactInvalidSetStateWarningDevTool');
```

Comment:
Normally we would just do `require('ReactInvalidSetStateWarningDevTool')`, so this seems weird.  I'm assuming you did this because `require('ReactInvalidSetStateWarningDevTool')` didn't work for you because you didn't have an `@providesModule` at the top of that file.

If you add the standard copyright header to `ReactInvalidSetStateWarningDevTool.js` and specify the `@providesModule`, we should be able to fix this line.